Catalog/Examples/Networking/Japx/NetworkingJapxViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  NetworkingJapxViewController.swift
+//  Catalog
+//
+//  Created by Mate Masnov on 02/12/2019.
+//  Copyright (c) 2019 Infinum. All rights reserved.
+//
+//  This file was generated by the 🐍 VIPER generator
+//
+
+import UIKit
+
+final class NetworkingJapxViewController: UIViewController {
+
+    // MARK: - Public properties -
+    
+    var presenter: NetworkingJapxPresenterInterface!
+    
+    // MARK: - Private properties -
+    
+    @IBOutlet private weak var _emailTextField: UITextField!
+    @IBOutlet private weak var _usernameTextField: UITextField!
+    @IBOutlet private weak var _passwordTextField: UITextField!
+    
+    @IBOutlet private weak var _createUserButton: UIButton!
+    @IBOutlet private weak var _getUserButton: UIButton!
+    @IBOutlet private weak var _updateUserButton: UIButton!
+    @IBOutlet private weak var _deleteUserButton: UIButton!
+    
+    @IBOutlet private weak var _scrollView: UIScrollView!
+    @IBOutlet private weak var _bottomConstraint: NSLayoutConstraint!
+    
+    private var _bottomHeight: CGFloat = 20
+    
+    // MARK: - Lifecycle -
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        updateView(with: .UserDoesNotExist)
+    }
+	
+    override func viewWillAppear(_ animated: Bool) {
+        _startObservingKeyboardEvents()
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        _stopObservingKeyboardEvents()
+    }
+    
+    // MARK: - Button actions -
+    
+    @IBAction func createUserActionHandler(_ sender: Any) {
```

Comment:
Can be private as well. Also maybe good styling approach would be to separate IBActions to extension